okapi-core/src/main/java/org/folio/okapi/util/TenantInstallOptions.java

```diff
@@ -103,4 +106,19 @@ public boolean getReinstall() {
   public void setReinstall(boolean reinstall) {
     this.reinstall = reinstall;
```

Comment:
`this` can be removed and other setters in the class don't use it.

Reply:
I don't understand what you mean.

Reply:
I didn't notice that the param name was the same as the variable name, so you do need `this`.